bot/exts/utils/utils.py

```diff
@@ -162,17 +162,29 @@ async def snowflake(self, ctx: Context, *snowflakes: Snowflake) -> None:
         if len(snowflakes) > 1 and await has_no_roles_check(ctx, *STAFF_ROLES):
             raise BadArgument("Cannot process more than one snowflake in one invocation.")
 
+        if not snowflakes:
+            raise BadArgument("At least one snowflake must be provided.")
+
+        embed = Embed(
+            colour=Colour.blue()
+        )
```

Comment:
I don't think the readability is poor as a single line.

Reply:
Agreed, this was a copy-paste oversight from when we also set the description here.